From: Dima Kogan <dima@secretsauce.net>
Date: Mon, 4 May 2020 18:00:40 -0700
Subject: I pass gccjit libraries to the linker in the order it wants them

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 90a86cd..35c4054 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3785,7 +3785,7 @@ AC_DEFUN
 COMP_OBJ=
 if test "${with_nativecomp}" != "no"; then
     emacs_save_LDFLAGS=$LDFLAGS
-    LDFLAGS="-lgccjit -ldl"
+    LIBS="-lgccjit -ldl"
     AC_RUN_IFELSE([libgccjit_smoke_test], [], [libgccjit_broken],
       [AC_LINK_IFELSE([libgccjit_smoke_test], [], [libgccjit_not_found])])
     LDFLAGS=$emacs_save_LDFLAGS
